### ISSUE TITLE
Keras Version

### ISSUE THREAD

**Issue by nattari** [aniket03/keras_compact_bilnear_CNN#1]
Thanks, Aniket for sharing the code.
While using your implementation, I get an error at the following line in _vgg_cbcnn.py_:
`x = merge(compact_bilinear_arg_list, mode=compact_bilinear, name='compact_bilinear', output_shape=output_shape_cb)`.
It gives a Typeerror (module object is not callable). By doing some search, I found that the new version of Keras has replaced merge with concatenate, but this did not help either. 
Do you have any insights?

**Comment by nattari** [comment]
<img width="720" alt="Screenshot 2019-06-19 at 18 24 17" src="https://user-images.githubusercontent.com/7510353/59783122-b8e5fd80-92bf-11e9-808d-8e9c47349f34.png">
On trying the implementation on a linux machine, I do not get version related error but I get another error. Please find attached the screenshot for the same.

**Comment by aniket03** [comment]
Hi @nattari Sorry for a late reply

1. Regarding point (1), yes layer `merge` has been replaced with `concatenate` and due to that you might be facing the error. For record I used the version `Keras==2.1.4` while training the model, which allows the usage of `merge` layer.

2. For point 2, I am not too sure, why you got this error. I guess you are using a `batch size of 64`, and have `200 classes` in your dataset. Could you pls check what is the shape of labels array that is being returned by `train_generator` or `val_generator`. Also if required (which IMO should not) you could try changing this line here to ensure that the target labels numpy array has the correct shape as required.

**Comment by liuqiang3** [comment]
I meet the same issue,when i using Lambda replaced merge ,the code can run.
